src/machine/machine.c

```diff
@@ -523,6 +524,36 @@ int machine_finalize(Machine *m) {
         return 0;
 }
 
+static bool machine_validate_unit(Machine *m) {
+        int r;
+        _cleanup_free_ char *unit = NULL;
+        _cleanup_free_ char *cgroup = NULL;
+
+        r = cg_pid_get_unit(m->leader, &unit);
+        if (!r && streq(m->unit, unit))
+                return true;
+
+        if (r == -ESRCH)
+                log_info("Machine unit is inactive, but the leader process is exited. "
```

Comment:
"inactive" → "active"
"is exited" → "has exited"
This should also be one line:
"Machine unit %s is active, but the leader process "PID_FMT" has exited."

Reply:
The warning message has been updated.